.php_cs.dist

```diff
@@ -37,6 +37,7 @@ return PhpCsFixer\Config::create()
         'semicolon_after_instruction' => true,
         'strict_comparison' => true,
         'strict_param' => true,
+        'method_chaining_indentation' => true,
```

Comment:
Please maintain this list sorted alphabetically: the rule should be inserted at line 26.